arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,881 @@
+          "accountResource"
+        ],
+        "summary": "Accounts_Get",
+        "description": "Gets the Visual Studio Team Services account resource details.",
+        "operationId": "accounts_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "resourceName",
+            "description": "Name of the resource.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the details of the Visual Studio Team Services account resource.",
+            "schema": {
+              "$ref": "#/definitions/AccountResourceGetResponse"
+            }
+         },
+          "404": {
+            "description": "The Visual Studio Team Services account does not exist."
+         }
+        },
+        "x-ms-examples": {
+            "getAccountResource": {
+                "$ref": "../examples/GetResource.json"
+            }
+        }   	
+      }
+    }
+  },
+  "definitions": {
+    "AccountResourceGetResponse": {
+      "description": "The response to an account resource GET request.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique identifier of the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "description": "Resource properties.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-client-flatten": true
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "type": {
+          "description": "Resource type.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "AccountResourceGroupGetResponse": {
+      "description": "The response to an account resource group GET request.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of resource details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccountResourceGetResponse"
+          }
+        }
+      }
+    },
+    "AccountResourcePutRequestData": {
```

Comment:
The ONLY use of `AccountResourcePutRequestData` model is to represent the body of Account Create operation.

1. But saw a bunch of properties `email, tenantId, upn`  present in this model which are ready only i.e. user cannot set them. Why they exists in this model?

2. Another set of properties those are marked readyOnly in this model are `subscriptionId, resourceGroupName, resourceName, resourceType, resourceProvider` (those are already exists in the operation path). User cannot set these properties in the body as well.

3. What is the purpose of operationType in the payload?